### ISSUE TITLE
any chance to have Persian or Arabic Fonts?

### ISSUE THREAD

**Issue by shahram7** [danmarshall/google-font-to-svg-path#39]
Hi There, I've tested just now your App for my name in Persian and I got disappointed. 
Is there any chance to get this characters as SVG?
Best Regards

**Comment by danmarshall** [comment]
Hello, can you give some example text and font name to help determine the cause of the issue?

**Comment by shahram7** [comment]
example text:
سلام دوست عزیز، ممنونم ازت
Font: Gulzar

**Comment by danmarshall** [comment]
Hi @shahram7, try going to [Google Fonts](https://fonts.google.com/) and search for fonts supported by language. [Here is a list](https://fonts.google.com/?subset=arabic&noto.script=Arab) of fonts in Arabic.

**Comment by danmarshall** [comment]
Yes, I believe it's font specific as to which languages are supported. Perhaps our UI should have a language dropdown?
